numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -285,6 +326,26 @@ static void
 
 /**end repeat**/
 
+/*  The following lines were generated using a slightly modified
+    version of code_generators/generate_umath.py and adding these
+    lines to defdict:
+
+defdict = {
+'inner1d' :
+    Ufunc(2, 1, None_,
+        r'''inner on the last dimension and broadcast on the rest \n"
+        "     \"(i),(i)->()\" \n''',
+          TD('ld'),
```

Comment:
something weird going on with the indentation here

Reply:
fixed, and aligned next block on 4-space-indent as well